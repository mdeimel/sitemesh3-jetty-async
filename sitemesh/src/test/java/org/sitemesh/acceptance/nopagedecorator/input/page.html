<html>
  <head>
    <title>Hello</title>
  </head>
  <body>

    <p>This page should not be decorated.</p>

    <sitemesh:decorate decorator="/inline-decorator.html">
      <p>But this content should be.</p>
    </sitemesh:decorate>

  </body>
</html>