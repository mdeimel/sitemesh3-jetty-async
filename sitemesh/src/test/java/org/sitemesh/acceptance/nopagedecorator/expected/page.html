<html>
  <head>
    <title>Hello</title>
  </head>
  <body>

    <p>This page should not be decorated.</p>

    <div class="inline">
      <p>But this content should be.</p>
    </div>

  </body>
</html>